src/libraries/Microsoft.Extensions.Logging.Console/src/WindowsLogConsole.cs

```diff
@@ -17,42 +17,108 @@ public WindowsLogConsole(bool stdErr = false)
         }
 
         private bool SetColor(ConsoleColor? background, ConsoleColor? foreground)
+        {
+            var backgroundChanged = SetBackgroundColor(background);
+            return SetForegroundColor(foreground) || backgroundChanged;
+        }
+
+        public bool SetBackgroundColor(ConsoleColor? background)
```

Comment:
Do these methods need to be public? I would think the only method that needs to be public is the methods that implement IConsole.